cucumber.sh

```diff
@@ -4,6 +4,25 @@
 node_modules/http-server/bin/http-server ./test/demo-app -p 9000 > /dev/null &
 server=$!
 #Run tests
-./node_modules/.bin/wdio test/wdio.conf.js
+#Use SPEC="..." to set path to .feature file to run tests from
+if [ "$SPEC" == "" ]; then
+    ./node_modules/.bin/wdio test/wdio.conf.js
```

Comment:
May be would be better to use something like 
`const specPart = $SPEC ? '  --spec "$SPEC"' : ''
./node_modules/.bin/wdio test/wdio.conf.js + specPart`
This will exclude wdio.conf.js copy-pasting + will give us an ability to add extra path parts.